domain/policy.go

```diff
@@ -234,6 +236,22 @@ func (r *RequirementTrigger) IsMatch(a *Appeal) (bool, error) {
 			}
 		}
 	}
+	if r.Expression != "" {
```

Comment:
What happens if both condition and expression get set in the policy ? Does any one take precedence ?

Reply:
each criteria within the RequirementTrigger will be evaluated, and the results combined with `AND`. So if there are more than one criteria provided (e.g. condition and expression) both evaluations have to be `true`